clients/java/src/test/java/io/camunda/zeebe/client/OAuthCredentialsProviderTest.java

```diff
@@ -360,7 +358,7 @@ private void mockCredentials(final String token, final String scope) {
             WireMock.post(WireMock.urlPathEqualTo("/oauth/token"))
                 .withHeader("Content-Type", equalTo("application/x-www-form-urlencoded"))
                 .withHeader("Accept", equalTo("application/json"))
-                .withHeader("User-Agent", matching("camunda-client-java/\\d+\\.\\d+\\.\\d+.*"))
```

Comment:
❓ Chnages by accident, I guess?

Reply:
Those changes are due to the copy/paste from stable/8.5